tezedge_state/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) SimpleStaking, Viable Systems and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+mod peer_address;
+pub use peer_address::{PeerAddress, Port};
+
+pub mod peer_crypto;
+pub use peer_crypto::PeerCrypto;
+
+pub mod acceptors;
+pub mod chunking;
+pub mod proposals;
+
+mod shell_compatibility_version;
+pub use shell_compatibility_version::*;
+
+mod effects;
+pub use effects::*;
+
+mod state;
+pub use state::*;
+
+mod tezedge_state_wrapper;
+pub use tezedge_state_wrapper::*;
+
+pub mod proposer;
+
+pub mod sample_tezedge_state;
+
+#[derive(Debug, Eq, PartialEq)]
+pub enum InvalidProposalError {}
```

Comment:
@binier 
wouldnt it be better to place it to `src/proposals/mod.rs` ?

Reply:
Yeah maybe, but I might just remove that as you can see, no error can occur just based on proposal.

Before I was validating time passed in proposal and erroring if proposal was outdated, but now each proposal contains `time_passed` instead, so we don't need that check anymore (and it is removed).

Reply:
@binier 
I see, so even better if we can remove that :)

Reply:
Am gonna keep it just in case we want to validate proposal somehow, but will move it to `src/proposals/mod.rs` as you suggested.